torchinductor/scheduler.py

```diff
@@ -458,6 +559,13 @@ def __init__(self, nodes):
                 assert False, node
         self.name_to_node = {node.get_name(): node for node in self.nodes}
 
+        if bool(os.environ.get("INDUCTOR_DEBUG", False)):
```

Comment:
Rename this to `INDUCTOR_SCHEDULER_GRAPH=1` or something like that for now.

We'll need a better system eventually (cc: @jansel ).

Reply:
renamed